### PR TITLE
change basemap

### PR DESCRIPTION
**Pull request by jodygarnett**

### CODE REVIEW

**Comment by aaime** [comment]
Only concern I have, is the map actually built with GeoServer? People looking at the URL are going to believe it is? :)

File: js/app.js [hunk 1]
```diff
@@ -23,31 +23,14 @@ function initMap() {
...
-            origin: ol.extent.getTopLeft(projectionExtent),
-            resolutions: resolutions,
-            matrixIds: matrixIds
-          }),
-          style: '_null',
-          extent: [-16643092.788385,4534550.2653588,-5988382.5431409,11207197.085613]
-        })
+         source: new ol.source.XYZ({
+           url: 'http://api.boundlessgeo.io/v1/basemaps/boundless/osm/{z}/{x}/{-y}.png'
+         })
```
Comment (tbarsballe) [inline js/app.js]: You removed a whole bunch of options here, some of which "ol.source.XYZ" still uses. In particular: projection, tilegrid, attributions.
See here for more details: http://openlayers.org/en/v3.8.2/apidoc/ol.source.XYZ.html

Comment (jodygarnett) [inline js/app.js]: Thanks, options restored

**Comment by jodygarnett** [comment]
I have been able to get this to work with:

* http://maps.boundlessgeo-dev.com/geoserver/gwc/

**Pull request closed without merging**